backend/src/question/question.controller.ts

```diff
@@ -0,0 +1,8 @@
+import { Controller } from "@nestjs/common";
+
+import { QuestionService } from "./question.service";
+
+@Controller("questions")
```

Comment:
Let's add a `GET` endpoint that retrieves a question by its ID. It should return the title, description, category, topics, and hints (which should be all the frontend needs to display in the question component).